Cargo.toml

```diff
@@ -21,43 +21,29 @@ reqwest = { version = "0.12.4", features = ["json", "stream"] }
 serde = "1.0.199"
 serde_json = "1.0.116"
 serde_with = "3.8.1"
-solana-account-decoder = "<1.17.0"
-solana-client = "<1.17.0"
-solana-sdk = "<1.17.0"
-solana-transaction-status = "<1.17.0"
-spl-token-2022 = { version = "0.9.0", features = ["no-entrypoint"] }
-spl-token = { version = "4.0.0", features = ["no-entrypoint"] }
-tokio = { version = "1.37.0", features = ["full"] }
+solana-account-decoder = "=2.0.3"
```

Comment:
Ill have to check out this branch and see if 2.0.3 is not going to cause shenanigans

the solana >=2 sdks were not compatible with the searcher client iirc, will lyk

Reply:
yup 
<img width="1686" alt="image" src="https://github.com/user-attachments/assets/bcb142fa-66d6-4c3d-9a9e-763fe2738aa5" />

honestly i might be dropping the direct raydium lib dependency since it sucks big time, it takes like 4 account calls to do the swap, comes with a lot of peer deps and jupiter can easily allows swapping most raydium tokens with better slipapge prot

Reply:
easier to monitor prices & pool events by using direct raydium though, yeh?

Reply:
yes and no, the lp layout you can carry over, rest of their lib is meh